{
    "response": "error",
    "cause": "INVALID_OAUTHSCOPE"
}